service/api_logic/teams_logic.py

```diff
@@ -8,27 +12,94 @@
 RESPONSE_KEY = "response"
 
 def get_teams(session):
-    blob_indexes = get_all_blob_indexes_from_db(session, TEAMS_JSON)
-    result = get_blob_data_for_all_sports(session, blob_indexes)
-    data = json.loads(result)
-    filtered_data = []
+    try:
+        blob_indexes = get_all_blob_indexes_from_db(session, TEAMS_JSON)
```

Comment:
variable naming

if we expect blob indexes for teams - then specify it: teams_blob_indexes or blob_indexes_teams

Do it for every variable you write (I'm not asking you to rewrite every variable name here,. Just telling you that for future)